charts/modules/apps/karpenter/values.yaml

```diff
@@ -161,7 +161,7 @@ crossplane-aws-iam:
                 - "sqs:ReceiveMessage"
               allowPassingInstanceRole:
                 Effect: Allow
-                Resource: 'arn:aws:iam::{{ .Values.global.awsAccountId }}:role/{{ include "common-gitops.names.release" . }}-node'
+                Resource: 'arn:aws:iam::{{ .Values.global.awsAccountId }}:role/{{ include "common-gitops.names.release" . }}*-node'
```

Comment:
I think we shoudl use `node*`. This way we can have easy filters for OPA policies